cli/src/main/kotlin/OrtMain.kt

```diff
@@ -189,7 +189,7 @@ class OrtMain : CliktCommand(name = ORT_NAME, invokeWithoutSubcommand = true) {
 
         val commandName = currentContext.invokedSubcommand?.commandName
         val command = commandName?.let { " '$commandName'" }.orEmpty()
-        val user = System.getProperty("user.name")
+        val user = System.getProperty("user.name").takeUnless { it == "?" } ?: Os.userHomeDirectory.name
```

Comment:
Commit message: Do you mean "how **it** may happen"?

Reply:
You forgot to remove the "to".

Reply:
Argh, fixed.